nomad/structs/structs.go

```diff
@@ -5553,6 +5569,37 @@ func (t *Task) Warnings() error {
 	return mErr.ErrorOrNil()
 }
 
+type Kind string
```

Comment:
Let's name it `TaskKind` as I could see us wanting to use "Kind" fields elsewhere (Consul named their similar field `ServiceKind`)